BBallStatsTrack/src/com/example/bballstatstrack/activities/AddPlayersToTeamsActivity.java

```diff
@@ -0,0 +1,205 @@
+            @Override
+            public void onClick( DialogInterface dialog, int which )
+            {
+                finish();
+            }
+        } );
+        builder.setNegativeButton( R.string.no, new DialogInterface.OnClickListener()
+        {
+            @Override
+            public void onClick( DialogInterface dialog, int which )
+            {
+                dialog.dismiss();
+            }
+        } );
+        builder.show();
+    }
+
+    @Override
+    protected void onCreate( Bundle savedInstanceState )
+    {
+        super.onCreate( savedInstanceState );
+        setContentView( R.layout.activity_new_team );
+        mHomeTeamFragment = new TeamFragment( true );
+        mAwayTeamFragment = new TeamFragment( false );
+        mHomeButton = ( Button ) findViewById( R.id.homeTeamButton );
+        mHomeButton.setOnClickListener( new FragmentButtonListener( mHomeTeamFragment ) );
+        mAwayButton = ( Button ) findViewById( R.id.awayTeamButton );
+        mAwayButton.setOnClickListener( new FragmentButtonListener( mAwayTeamFragment ) );
+        mProceedButton = ( Button ) findViewById( R.id.proceedButton );
+        mProceedButton.setOnClickListener( new ProceedButtonListener() );
+        replaceCurrentFragment( mHomeTeamFragment );
+        updateView( mHomeTeamFragment );
+    }
+
+    private Fragment getCurrentFragment()
+    {
+        FragmentManager manager = getFragmentManager();
+        return manager.findFragmentById( R.id.fragmentContainer );
+    }
+
+    private void replaceCurrentFragment( Fragment fragment )
+    {
+        FragmentManager manager = getFragmentManager();
+        FragmentTransaction transaction = manager.beginTransaction();
+        transaction.replace( R.id.fragmentContainer, fragment );
+        transaction.commit();
+    }
+
+    private void updateView( Fragment current )
+    {
+        if( mHomeTeamFragment.equals( current ) )
+        {
+            mHomeButton.setEnabled( false );
+            mAwayButton.setEnabled( true );
+        }
+        else if( mAwayTeamFragment.equals( current ) )
+        {
+            mHomeButton.setEnabled( true );
+            mAwayButton.setEnabled( false );
+        }
+        else
+        {
+            mHomeButton.setEnabled( true );
+            mAwayButton.setEnabled( true );
+        }
+    }
+
+    private class FragmentButtonListener implements OnClickListener
+    {
+        Fragment mFragment;
+
+        public FragmentButtonListener( Fragment fragment )
+        {
+            mFragment = fragment;
+        }
+
+        @Override
+        public void onClick( View v )
+        {
+            if( mFragment.equals( getCurrentFragment() ) )
+            {
+                return;
+            }
+            replaceCurrentFragment( mFragment );
+            updateView( mFragment );
+        }
+    }
+
+    private class ProceedButtonListener implements OnClickListener
+    {
+        public boolean hasCompleteTeamNames()
```

Comment:
At first read, "hasCompleteTeamNames" name could indicate that both teams have submitted their names. please rename the method. thanks.